index.html

```diff
@@ -115,9 +114,123 @@
         </div>
       </div>
     </div> 
-    
-    <!-- LATEST FUNDING -->
 
+    <!-- Latest News -->
```

Comment:
## Improve the `general layout` of the page
- [ ] The `general layout` of the page from top to bottom are not aligned as viewed or compared with the original page. You should note that the first main three articles must not cover the same width as that of the rest of the contents of the page. The width is much wider in this section as the rest of the section gets to be uniform, lesser and centred. Have a look at the comparison below for inspiration. 👇 
- [ ] Be mindful of the spacing around elements and the font sizes as well. they appear much bolder on the source page.

|[Source Page](https://perma.cc/M5ZV-Q2D6)|[Clone Page](https://raw.githack.com/jmontanholi/ProjectTwoMilestoneTwo/categories/index.html)|
|:--:|:--:|
|![tnw](https://user-images.githubusercontent.com/22345961/109206523-03b85800-77b9-11eb-8a40-18ad60aa574f.png)|![tnw-clone](https://user-images.githubusercontent.com/22345961/109206560-103cb080-77b9-11eb-9059-9d8c2b8d12a8.png)|